apps/ai-content-generator/src/index.tsx

```diff
@@ -6,7 +6,7 @@ import { SDKProvider } from '@contentful/react-apps-toolkit';
 import App from './App';
 import LocalhostWarning from '@components/common/LocalhostWarning';
 import { QueryClientProvider } from 'react-query';
-import queryClient from '@/react-query';
+import queryClient from '@react-query/index';
```

Comment:
is this meant to pull from the local module or the installed package?

Reply:
I was having an issue with the alias directly resolving to the module. Adding an alias that points directly to it, I am able to fix this.

I'll add this to the next PR